index.html

```diff
@@ -1 +1,138 @@
+          <h4 class="signup-subtitle">Další Hradecká NodeSchool bude 15. ledna 2014</h4>
+        </div>
+      </div>
+      <div class="row">
+        <!-- START: Signup Form Bottom -->
+        <div class="eight columns centered">
+          <div class="signup-form">
+            <a href="http://srazy.info/nodeschool/4891" class="btn-side animate-on-scroll" data-scrollanimation="fadeInLeft"><i class="icon-ticket"></i>Přihlaš se!</a>
+          </div>  
+        </div>
+        <!-- /END: Signup Form Bottom -->
+      </div>
+    </section>
+    <!-- /END: Signup Section -->
+  
+  <section class="site-section">
+    <div class="row">
+      <div class="twelve columns animate-on-scroll" data-scrollanimation="fadeDownIn">
+        <h3>Otázky? Názory? Nabídky spolupráce?</h3>
+        <h4>Založ issue na <a href="https://github.com/nodeschool/hradec-kralove">GitHubu</a> nebo se zapoj do diskuze.</h4>
+          <p> Pokud ještě nepoužíváš GitHub, založ si <a href="https://github.com/join">účet</a> a podívej 
+          se na <a href="http://nodeschool.io/#git-it">Git-it</a> cvičení. Ukáže ti základy práce s Gitem.</p>
+      </div>
+    </div>
+  </section>
+  </main>
+  <!-- START: Site Footer -->
+  <footer role="contentinfo" class="site-footer">
+    <div class="row footer-info">
+      <!-- START: Copyright Notice -->
+      <div class="six columns">
+        <p class="footer-copyright">HTML, CSS, images, and design are based on a design 
+          <a href="http://github.com/pikkupanda/pikkupanda.github.io">forked from pikkupanda.de</a> which was , in turn,
+          <a href="https://github.com/maxogden/csvconf.com">forked from csvconf.com</a> which was, in turn, 
+          <a href="https://github.com/sudomesh/peoplesopen-front">forked from PeoplesOpen.Net</a> which was, in turn, 
+          forked from <a href="http://martini.codegangsta.io/">Martini’s template</a>, reused here under the MIT License. 
+          Based (forked) on <a href="https://github.com/nodeschool/berlin">Berlin Nodeschool page</a>. 
+          The background picture is <a href="https://creativecommons.org/licenses/by-sa/2.0/">CC BY-SA 2.0</a> by <a href="https://www.flickr.com/photos/korom/4518205690">János Korom</a>
+          The icons used are <a href="http://creativecommons.org/licenses/by-sa/3.0/">CC BY-SA 3.0</a> by  <a href="http://www.entypo.com/">Daniel Bruce</a>.</p>
+      </div>
+      <!-- /END: Copyright Notice -->
+      <div class="six columns">
+        <h3> Sponzoři </h3>
+        <a target="_blank" href="http://strongloop.com/">
+          <img src="img/sponsors/strongloop.png" alt="StrongLoop"/>
+        </a>
+        <a target="_blank" href="http://coolworking.cz/">
+          <img src="img/sponsors/coolworking.png" alt="CoolWorking"/>
+        </a>
+        <a target="_blank" href="http://www.gug.cz/cs/gdg/skupiny/hradec-kralove">
+          <img src="img/sponsors/gug.png" alt="GUG Hradec Králové"/>
+        </a>
+      </div>
```

Comment:
Šlo by to prosím rozdělit na dvě části?

První - "Sponzoři" - bude obsahovat jenom StrongLoop.
Druhá - "Partneři" - pro CoolWorking a GUG.